mix.exs

```diff
@@ -30,7 +31,7 @@ defmodule Lob.Mixfile do
   defp deps do
     [
       {:bypass, "~> 0.8", only: :test},
-      {:credo, "~> 0.8", only: [:dev, :test], runtime: false},
+      {:credo, "~> 0.9.0-rc1", only: [:dev, :test], runtime: false},
```

Comment:
Any reason why we're using an RC dependency? I'd rather stick with 0.8 until 0.9 is released.

Reply:
Ah I just chose the latest version recommended on the repo page. `0.8` has a bug with Elixir `1.6.x`, I can hunt around for another version.

Reply:
Sweet, I think `v0.8.10` fixed the bug.